src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java

```diff
@@ -587,16 +587,16 @@ public int getPhysicalProcessorCount() {
     @Override
     public String toJSON() {
         StringBuilder sb = new StringBuilder("{");

```

Comment:
There are dozens. Jackson is the most common. 

The overhead to create the dictionaries is more lines of code than building the strings. And this way avoids another dependency.